### ISSUE TITLE
[VisDashboard] Size problem

### ISSUE THREAD

**Issue by chseifert** [EEXCESS/visualization-widgets#37]
From the feedback we received through the chrome webstore (from a user):
"Hi, when I click on the dashboard I usually get the feedback "This visualization requires a larger screen area. Minimum width 750px and minimum height 200px." even if I use my big screen with 1920x1200.
Or is the dashboard not ready yet?"
- To make it clear (that a huge monitor is not enough) - could we just rephrase it to "currently the window is of size AxB" (to indicate how much space is additionally needed)

**Comment by gerwald** [comment]
As it might not be so clear for all users, that the window is resizeable, wouldn't it be an idea, to automatically set the window to 750px, after the user clicks on the dashboard button?

I think we have spoken about this at the hackathon, but I am not sure any more, why we didn't do that / remove this functionality...

**Comment by chseifert** [comment]
I think the result of the discussion was that because we allow for manual resizing of the window, we assume that once the user has resized the window s/he likes it that way. But I think we agreed also that we change it to "large enough" if not manual resizing has occurred before. Assign to Jörg to give us his view (maybe he remembers differently).

**Comment by mgrani** [comment]
maybe there is a middle way: add some 'resizing' buttons: one to maximize, one for max width, half height. Could be left to the 'x" button

**Comment by chseifert** [comment]
additional question to Gerwald: Did you receive the feedback as well? It was sent via the feedback form, but I don't know who should it receive. I did get it only, because it was explicitly sent by mail to me.

**Comment by gerwald** [comment]
That the feedback form is sent via the logging framework afaik, and should not be displayed, when the screen size is too low (this is a bug).
Thus, I only get the feedback, if I ask for the whole logging result text file (what I haven't done so far)

**Comment by schloett** [comment]
I like the idea of resizing buttons. Maybe in addition, we could introduce a message event? Something like `eexcess.tooSmall` with the minimal dimensions appended. This way, we would have a generic approach (required dimensions for the FacetScape differ from Dashboard). I would imagine something like:

``` javascript
if(tooSmall) {
  var dimensions = {
    width:800, // width in pixels
    height:300 // height in pixels
  }
  var msg = {
    event: 'eexcess.tooSmall',
    data: dimensions
  }
  window.top.postMessage(msg, '*');
} 
```

From the viewpoint of the chrome-extension, this event might cause a popup with a notification about the contentArea being to small, a link to resize it and two (pre-checked) checkboxes for not showing the popup again and making the resize the default behavior (which can be changed via the settings page).

**Comment by schloett** [comment]
Created a new issue for the resizing buttons #40. Any comments on the proposed event?

**Comment by gerwald** [comment]
FYI: I have just implemented the "tooSmall" event, so the extension could react on this (eg when the dashboard is opened the first time)
